### DISCUSSION TITLE
Issue with Deployment

### DISCUSSION THREAD

**Discussion by srenna** [microsoft/AI-Chat-App-Hack#124] [Q&A] [answered]
After a few successful pushes, I can't get the image to push successfully to the container instances. 
<img width="1218" alt="Screenshot 2024-02-12 at 9 01 42 PM" src="https://github.com/microsoft/AI-Chat-App-Hack/assets/133055434/6f6fe0b1-c692-481c-9073-0cb003e04f81">

**Comment by pamelafox** [comment, marked as answer]
Hi @srenna, I can't tell what the error might be from that screenshot. I generally check both the deployment logs and container logs when I'm debugging container apps. It's not a strict requirement to have a deployed URL for submission, feel free to just include the repo URL and a video if the deploy isn't working tonight.

**Reply by srenna** [reply]
Thank you @pamelafox! We got it working. 
[http://hackapp.eastus.azurecontainer.io/](url)
